README.md

```diff
@@ -2,4 +2,13 @@
 
 Python interface for the Geospatial Common Data Library.
 
+To set up a jupyter notebook kernel for pygcdl, run the following commands:
 
+module load python_3
```

Comment:
I think you might need to make a code block in markdown for this section to look nicely rendered.

Reply:
Fixed on fork